packages/server/src/synchronizer/pipeline/rules/pages/index.ts

```diff
@@ -52,7 +52,7 @@ const createRulePages = ({config, errors, getInputCache}: RuleArgs) => {
           ? 'Warning: You have tried to put an api route inside a pages directory:'
           : 'Warning: You have tried to put api routes inside a pages directory:'
 
-      const secondary = 'All api routes should be in their own directory.'
+      const secondary = 'All api routes should be in their own directory (/app/api)'
```

Comment:
This sounds like it must be at the root `app` level where as it would be fine to be in a context. `app/products/api`

Reply:
What about that?

Api routes should be in their own directory (outside of pages folder):
- Context-specific routes can be nested in a context directory (app/products/api) 
- Global Api routes such as webhooks can be at root level (app/api)